test/macros.jl

```diff
@@ -274,12 +274,16 @@ end
     @testset "Adjoints" begin
         model = Model()
         @variable(model, x[1:2])
-        obj = @objective(model, Min, x' * x)
-        @test obj == sum(x.^2)
-        cref = @constraint(model, x' * x <= 1)
+        Q = [1.0 0.0; 0.0 1.0]
+        obj = @objective(model, Min, x' * Q * x)
```

Comment:
`x' Q x` doesn't trigger the bug, we need at least `x' ones(2, 2) x`. If every factor is just a symbol it won't trigger the bug